tests/fixture_valid.js

```diff
@@ -80,13 +80,55 @@ module.exports.checkFixture = function checkFixture(fixture, usedShortNames=[])
       }
 
       for (const ch of mode.channels) {
-        if (!(ch in fixture.availableChannels) && ch !== null) {
-          result.errors.push({
-            description: `channel '${ch}' referenced from mode '${modeShortName}' (#${i}) but missing.`,
-            error: null
-          });
-        }
         usedChannels.push(ch);
+
+        if (ch !== null) {
+          if (!(ch in fixture.availableChannels) && ch !== null) {
```

Comment:
You don't need to double-check `ch !== null`. Then, please also swap if and else to get rid of the extra negation.